src/deps_deploy/deps_deploy.clj

```diff
@@ -6,6 +6,11 @@
             [clojure.java.io :as io]
             [clojure.data.xml :as xml]))
 
+
+(aether/register-wagon-factory! "s3p" #(org.springframework.build.aws.maven.PrivateS3Wagon.))
```

Comment:
I think I'd like for this to be `:imported` so we can say 
`((aether/register-wagon-factory! "s3p" #(PrivateS3Wagon.))`
Ditto for the following line.